### PR TITLE
LRQA-68208 Fix FormsSubmissions#SubmitButtonTextCanBeCustomizedWithSpecialCharacters Test

### PR DESCRIPTION
**Pull request by fmarcio**

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase [hunk 1]
```diff
@@ -548,6 +548,8 @@ definition {
 
 		Form.configureSubmitLabel(submitLabel = "!@#$%^&*()+?");
 
+		Button.click(button = "Done");
+
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:552]: @fmarcio This [commit](https://github.com/liferay-forms/liferay-portal/commit/ecf9cef5403341cea1e70150fa82683b280e7329) recently added to the `configureSubmitLabel` macro the `Button.clickDone();`.
Can you check if your change is still needed? If not, can you please remove it? Thanks!

Comment (fmarcio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase]: > @fmarcio This [commit](https://github.com/liferay-forms/liferay-portal/commit/ecf9cef5403341cea1e70150fa82683b280e7329) recently added to the macro `configureSubmitLabel` the `Button.clickDone();`.
> Can you check if your change is still needed? If not, can you please remove it? Thanks!

@carolmariaabb not needed anymore. Fixed! :-)

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**